### DISCUSSION TITLE
Support server proxy for guests

### DISCUSSION THREAD

**Discussion by alcorwalter** [vmware/packer-examples-for-vsphere#804] [Ideas]
There were 2 issues about that but they have been closed automatically, without any debate. This is not an uncommon occurrence in a professional setting. 
Are some guests systems difficult or impossible to configure? I don't have the necessary staging environment to deal with it, but it looks like it's not difficult to configure for some systems.

https://github.com/vmware-samples/packer-examples-for-vsphere/issues/228
https://github.com/vmware-samples/packer-examples-for-vsphere/issues/232

**Comment by tenthirtyam** [comment]
#232 is reopened and in the backlog.
